lib/spectacles/schema_statements/postgresql_adapter.rb

```diff
@@ -7,11 +7,10 @@ module PostgreSQLAdapter
 
       def views(name = nil) #:nodoc:
         q = <<-SQL
-              SELECT t.table_name, t.table_type
-                FROM information_schema.tables AS t
-          INNER JOIN pg_class AS c ON c.relname = t.table_name
-               WHERE t.table_schema = ANY(current_schemas(false))
-                 AND t.table_type = 'VIEW'
+              SELECT t.table_name
```

Comment:
This looks much more straightforward, but is there a minimum version of PG this works with?

Reply:
I see the `information_schema.views` as far back as Postgres 7.4 https://www.postgresql.org/docs/7.4/infoschema-views.html, unless I'm interpreting it wrong.